imgee/__init__.py

```diff
@@ -12,20 +11,18 @@
 import coaster.app
 from ._version import __version__
 
-
 version = Version(__version__)
 app = Flask(__name__, instance_relative_config=True)
 lastuser = Lastuser()
-celery = Celery()
 
 assets['imgee.css'][version] = 'css/app.css'
 
 from . import models, views
 from .models import db
-from .api import api
-from .async import TaskRegistry
+from .tasks import TaskRegistry
+
+registry = TaskRegistry(os.getenv('ENV', 'production'))
```

Comment:
Can I set this inside the `init_for(env)` function and then use env? `TaskRegistry` has `name='default'` set by default. I wanted to use something more verbose for the redis keys. hence did this.